src/main/java/com/google/devtools/build/lib/bazel/repository/GitCloner.java

```diff
@@ -181,6 +189,15 @@ public static SkyValue clone(Rule rule, Path outputDirectory, EventHandler event
     return new HttpDownloadValue(descriptor.directory);
   }
 
+  private static void setProxyIfNeeded() throws IOException {
+	if(!"".equalsIgnoreCase(System.getenv("HTTP_PROXY"))){
```

Comment:
This is duplicating the code in HttpDownloader, might as well move them both into ProxyHelper.